client/luigi-client.js

```diff
@@ -39,205 +37,220 @@
-        var relativePath = path[0] !== '/';
-        var navigation = {
-          msg: 'luigi.navigation.open',
-          sessionId: sessionId,
-          params: Object.assign(
-            { link: path },
-            { relative: relativePath },
-            options
-          )
-        };
-        window.parent.postMessage(navigation, '*');
+        nodeParams: {},
+        errorSkipNavigation: false,
+        fromContext: null,
+        fromClosestContext: false
       };
 
       return {
+        /** @lends linkManager */
         /**
-         * Navigates to the given path in the hosting Luigi app.
+         * Navigates to the given path in the hosting Luigi app. Contains either a full absolute path or a relative path without a leading slash that uses the active route as a base. This is a classical navigation.
          * @param {string} path path to be navigated to
          * @param {string} sessionId  current Luigi sessionId
-         * @param {boolean} preserveView open route in a new view window to goBack to last state afterwards
+         * @param {boolean} preserveView Preserve a view by setting it to true. It keeps the current view opened in the background and opens the new route in a new frame. You must use the function {@link #goBack goBack()} to navigate back afterwards. You can use this feature at unlimited levels.The preserved views are discarded as soon as the standard {@link #navigate navigate()} function is in use in place of {@link #goBack goBack()} .
+         * @example 
+         * LuigiClient.linkManager().navigate('/overview')
+         * LuigiClient.linkManager().navigate('users/groups/stakeholders')
+         * LuigiClient.linkManager().navigate('/settings', null, true) // preserve view
          */
         navigate: function navigate(path, sessionId, preserveView) {
           if (options.errorSkipNavigation) {
-            options.errorSkipNavigation = undefined;
+            options.errorSkipNavigation = false;
             return;
           }
           options.preserveView = preserveView;
-          _navigate(sessionId, path);
+          var relativePath = path[0] !== '/';
+          var navigationOpenMsg = {
+            msg: 'luigi.navigation.open',
+            sessionId: sessionId,
+            params: Object.assign(
+              { link: path, relative: relativePath },
+              options
+            )
+          };
+          window.parent.postMessage(navigationOpenMsg, '*');
         },
 
         /**
-         * Sets the current navigation context, which is then be used by navigate function
-         * Usage: linkManager.fromContext("currentTeam").navigate("path")
-         * @param {object} navigationContext
-         * */
+         * Sets the current navigation context to that of a specific parent Node that has the `navigationContext` field declared in its navigation configuration. This navigation context is then used by navigate function.
```

Comment:
Can we link `navigationContext` to the configuration documentation?

Reply:
Linked to `navigation-configuration.md#nodes`